libs/dh/market-participant/feature-market-participant/src/components/edit.component.ts

```diff
@@ -146,16 +141,16 @@ export class DhActorsEditActorModalComponent extends WattTypedModal<DhMarketPart
   }
 
   save() {
-    const { departmentEmail, departmentName, departmentPhone, name } = this.actorForm.getRawValue();
+    const { departmentEmail, departmentName, departmentPhone, name } = this.form.getRawValue();
 
```

Comment:
suggestion: make sure the user is navigated to details on success. As it is right now, it seems like the modal will just close.